schema.sql

```diff
@@ -0,0 +1,287 @@
+CREATE TABLE
+  account (
+    id VARCHAR PRIMARY KEY,
+    total_donations_usd DECIMAL(10, 2),
+    total_matching_pool_allocations_usd DECIMAL(10, 2),
+    donors_count INT
+  );
+
+CREATE TABLE
+  list (
+    id VARCHAR PRIMARY KEY,
+    owner_id VARCHAR NOT NULL,
+    name VARCHAR NOT NULL,
+    description TEXT,
+    default_registration_status ENUM(
+      'Pending',
+      'Approved',
+      'Rejected',
+      'Graylisted',
+      'Blacklisted'
+    ) NOT NULL,
+    FOREIGN KEY (owner_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  list_admin (
+    list_id INT NOT NULL,
+    admin_id VARCHAR NOT NULL,
+    PRIMARY KEY (list_id, admin_id),
+    FOREIGN KEY (list_id) REFERENCES list (id),
+    FOREIGN KEY (admin_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  list_registration (
+    id SERIAL PRIMARY KEY,
+    registrant_id VARCHAR NOT NULL,
+    status ENUM(
+      'Pending',
+      'Approved',
+      'Rejected',
+      'Graylisted',
+      'Blacklisted'
+    ) NOT NULL,
+    submitted_at TIMESTAMP NOT NULL,
+    updated_at TIMESTAMP,
+    registrant_notes TEXT,
+    admin_notes TEXT,
+    tx_hash VARCHAR,
+    FOREIGN KEY (registrant_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  pot_factory (
+    id VARCHAR PRIMARY KEY,
+    owner_id VARCHAR NOT NULL,
+    deployed_at TIMESTAMP NOT NULL,
+    source_metadata VARCHAR NOT NULL,
+    protocol_fee_basis_points INT NOT NULL,
+    protocol_fee_recipient_account VARCHAR NOT NULL,
+    require_whitelist BOOLEAN NOT NULL,
+    FOREIGN KEY (owner_id) REFERENCES account (id),
+    FOREIGN KEY (protocol_fee_recipient_account) REFERENCES account (id)
+  );
```

Comment:
add `FOREIGN KEY (id) REFERENCES account (id),`

Reply:
`account_id` is used as the id since it's a unique, hence why i removed the separate account id field,